package.json

```diff
@@ -58,6 +58,7 @@
     "delay": "6.0.0",
     "electron-log": "4.4.8",
     "fast-deep-equal": "3.1.3",
+    "file-saver": "2.0.5",
```

Comment:
The only issue I see is that this package seems unmaintained. (Last commit is from 4+ years ago). But I guess this is easy to replace if this becomes an issue in the future due to version incompatibilty or else. Otherwise :100: Great job!

Reply:
Thanks!